cmd/forklift-api/BUILD.bazel

```diff
@@ -22,7 +22,6 @@ go_library(
 go_binary(
     name = "forklift-api",
     embed = [":forklift-api-lib"],
-    static = "on",
```

Comment:
why was it on? also, isn't it the default

Reply:
there was an issue related to network with the version of Go we used in 2.4 and that was a workaround that issue

Reply:
> also, isn't it the default

ah and about the default settings, [it's disabled by default](https://github.com/bazelbuild/rules_go/blob/8c014fd0798b1c2965d4dc97e11089bc1d43490a/go/modes.rst#build-settings)

Reply:
hmm go's default is static unless CGO is enabled

Reply:
the network issue is resolved with netdns=go no?

Reply:
yeah, looking at git log, it wasn't part of the fix for that network issue but part of the original changes to create an image for forklift-controller using bazel - it was in 09/22, I don't recall why it was done this way
anyway, it's not needed anymore - without it the provider is added but now there's an issue with mounting the NFS volume to the server pod, I hope it's due to v4...